lib/presets.js

```diff
@@ -27,9 +27,42 @@ const path = require('path');
 const updateJSON = require('update-json');
 const fs = require('fs');
 const semver = require('semver');
-const replace = require('replace-in-file');
+const replaceInFile = require('replace-in-file');
 const markdown = require('./markdown');
 
+/**
+ * @summary Replaces the contents of a file using a regular expression
+ * @function
+ * @private
+ *
+ * @param {String} file - target filename
+ * @param {RegExp} pattern - pattern used in str.replace
+ * @param {String} replacement - replacement used in str.replace
+ * @param {Function} callback - callback (error)
+ */
+const replace = (file, pattern, replacement, callback) => {
+  async.waterfall([
+    (done) => {
+      fs.readFile(file, 'utf8', done);
+    },
+
+    (contents, done) => {
+      if (pattern.test(contents)) {
+        done(null, contents);
+      } else {
+        done(new Error(`Regex does not match ${file}`));
+      }
+    },
+
+    (contents, done) => {
+      const updated = contents.replace(pattern, replacement);
+      done(null, updated);
+    },
+
+    _.partial(fs.writeFile, file)
```

Comment:
Given that we explicitly set a file-encoding for `readFile`, should we also explicitly set a file-encoding for `writeFile` ?

Reply:
The encoding defaults to 'utf-8' when writing. It is different than readFile..

Reply:
'utf8', sorry